tools/js2c.cc

```diff
@@ -54,7 +54,21 @@ int GetStats(const char* path, std::function<void(const uv_stat_t*)> func) {
   return r;
 }
 
+constexpr bool EndsWith(const std::string& str,
```

Comment:
I *do not* recommend you change something but it is worth knowing that this function will become unnecessary with C++20: https://en.cppreference.com/w/cpp/string/basic_string/ends_with

Reply:
Reverted the changes.